react/features/app/components/App.native.js

```diff
@@ -140,6 +140,7 @@ export class App extends AbstractApp {
         }
 
         dispatch(updateSettings(this.props.userInfo || {}));
+        dispatch(updateSettings(this.props.settings || {}));
```

Comment:
Why is this necessary?

Reply:
Those are passed as config overrides, and the config middleware takes care of the rest.

Check the asProps methods on the native SSK to see how those are passed.